examples/asitvd.html

```diff
@@ -31,7 +31,7 @@
     <script src="default.js"></script>
     <script src="../utils/watchwatchers.js"></script>
     <script>
-      var ngeoModule = angular.module('ngeo');
+      let ngeoModule = angular.module('ngeo');
```

Comment:
The examples must also be viewable with ES5 browsers so you can not use let here and in any html files.